data/OR/incentives.json

```diff
@@ -0,0 +1,962 @@
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_savingsWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.5,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$1.50 per sqft of attic insulation for manufactured homes. Income-qualified customers. Existing insulation must be R-18 or less."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-27",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.25,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "$1.25 per sqft of attic insulation for manufactured homes. Existing insulation must be R-18 or less. Must insulate to R-30 or greater."
+    }
+  },
+  {
+    "id": "OR-28",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_savingsWithinReach",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.5,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$1.50 per sqft of attic insulation. Income-qualified customers. Existing insulation must be R-18 or less. Must insulate to R-38 or greater."
+    },
+    "low_income": "or-energy-trust-of-oregon"
+  },
+  {
+    "id": "OR-29",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 1.25,
+      "unit": "square_foot"
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$1.25 per sqft of attic insulation for manufactured homes. Existing insulation must be R-18 or less. Must insulate to R-30 or greater."
+    }
+  },
+  {
+    "id": "OR-30",
+    "authority_type": "state",
+    "authority": "or-energy-trust-of-oregon",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "other",
+    "program": "or_energyTrustOfOregon_energyTrustOfOregon",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 250
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$250 discount on qualifying central air conditioners. Outdoor unit is required to achieve a minimum nominal rating of SEER2 15.20 or greater."
```

Comment:
We don't generally do air conditioners, since they're already electric and we want people to get heat pumps instead.

Add a column called `Omit from API?` to the spreadsheet and put a `TRUE` value there for this row. When you rerun the script, it should not have this incentive anymore.

Alternatively, you could delete it entirely, though I suppose it might be nice to keep around in case we change our minds.